rust/lib/src/lib.rs

```diff
@@ -37,14 +37,19 @@ fn construct_uri_load_config(
     uri: String,
     data_dir: String,
 ) -> Result<(zingoconfig::ZingoConfig, u64), String> {
-    let server = construct_server_uri(Some(uri));
+    let lightwalletd_uri = construct_lightwalletd_uri(Some(uri));
 
-    let (mut config, latest_block_height) = match zingolib::load_clientconfig(server, None) {
-        Ok((c, h)) => (c, h),
+    let mut config = match zingolib::load_clientconfig(
+        lightwalletd_uri.clone(),
+        None,
+        zingoconfig::ChainType::Mainnet,
```

Comment:
I'd like to get #276 in ASAP, to get a better solution than hard-coded mainnet...transaction fees are about to increase when zip317 lands, which we can bandaid by hardcoding a fee of the new min-fee (10000), and then maybe when we minor-release that we can get #276 in if it's ready? I'm mostly thinking out loud.